python/ray/accelerators/accelerators.py

```diff
@@ -0,0 +1,3 @@
+from ray.ray_constants import RESOURCE_CONSTRAINT_PREFIX
+
+NVIDIA_TESLA_V100 = f"{RESOURCE_CONSTRAINT_PREFIX}V100"
```

Comment:
Please rename this to just "V100", like in the tfc.AcceleratorType enum. The prefix should be added by code instead of in the enum.